libs/dh/wholesale/feature-calculation-steps/src/lib/dh-wholesale-calculation-steps.component.html

```diff
@@ -72,37 +103,55 @@ <h2 style="display: flex; align-items: center">
       <main>
         <ul class="steps" *ngIf="batch && gridArea">
           <li>
-            <dh-wholesale-production-per-gridarea
-              [batch]="batch"
-              [gridArea]="gridArea"
-            ></dh-wholesale-production-per-gridarea>
+            <watt-card (click)="openDrawer(1)">
+              <h4>
+                <watt-badge size="large" class="watt-space-inline-m"
+                  >1</watt-badge
+                >
+                {{ t("processStepResults.aggregatedProductionPerGridArea") }}
+              </h4>
+              <watt-button variant="text" icon="openInNew">
+                {{ t("calculationSteps.open") }}
+              </watt-button>
+            </watt-card>
           </li>
+          <!-- <li>
+            <watt-expandable-card>
+              <watt-expandable-card-title>
+                <watt-badge size="large">2</watt-badge>
+                {{
+                  t("processStepResults.aggregatedConsumptionPerEnergySupplier")
+                }}
+              </watt-expandable-card-title>
+              <button (click)="openDrawer(2)">Open drawer</button>
+            </watt-expandable-card>
+          </li> -->
```

Comment:
🎵Darling, you got to let me know
Should I stay or should I go?🎵 

`- The Clash`

Reply:
Old school feature flagging 😎